libsyclinterface/tests/test_sycl_device_aspects.cpp

```diff
@@ -97,7 +97,7 @@ auto build_gtest_values(const std::array<std::pair<T1, T2>, N> &params)
 auto build_params()
 {
     constexpr auto param_1 = get_param_list<const char *>(
-        "opencl:gpu", "opencl:cpu", "level_zero:gpu", "host");
```

Comment:
Not for this PR, but the design where a list of filter string is used is an anti-pattern. I got rid of a similar pattern in numba-dpex unit tests last year. All tests should run on default device and we can control the default device using the ONEAPI_DEVICE_SELECTOR environment variable.